apps/condo/domains/ticket/components/TicketId/TicketDeadlineField.tsx

```diff
@@ -18,39 +23,43 @@ export const TicketDeadlineField: React.FC<TicketDeadlineFieldProps> = ({ ticket
     const ToCompleteMessage = intl.formatMessage({ id: 'ticket.deadline.ToComplete' }).toLowerCase()
     const LessThenDayMessage = intl.formatMessage({ id: 'ticket.deadline.LessThenDay' }).toLowerCase()
     const OverdueMessage = intl.formatMessage({ id: 'ticket.deadline.Overdue' }).toLowerCase()
+    const CompletedEarlierMessage = intl.formatMessage({ id: 'ticket.deadline.CompletedEarlier' }).toLowerCase()
+    const CompletedLessThenDayMessage = intl.formatMessage({ id: 'ticket.deadline.CompletedLessThenDay' }).toLowerCase()
+    const CompletedLateMessage = intl.formatMessage({ id: 'ticket.deadline.CompletedLate' }).toLowerCase()
 
     const ticketDeadline = ticket.deadline ? dayjs(ticket.deadline) : null
     const getTicketDeadlineMessage = useCallback(() => {
         if (!ticketDeadline) return
 
+        const isCompleted = isCompletedTicket(ticket)
         const deadlineType = getDeadlineType(ticket)
         const { dayDiff, overdueDiff } = getHumanizeDeadlineDateDifference(ticket)
 
         switch (deadlineType) {
             case TicketDeadlineType.MORE_THAN_DAY: {
                 return (
                     <Typography.Text type='warning' strong>
-                        ({ToCompleteMessage.replace('{days}', dayDiff)})
+                        ({!isCompleted ? ToCompleteMessage.replace('{days}', dayDiff) : CompletedEarlierMessage.replace('{days}', dayDiff)})
                     </Typography.Text>
                 )
             }
             case TicketDeadlineType.LESS_THAN_DAY: {
                 return (
                     <Typography.Text type='warning' strong>
-                        ({LessThenDayMessage})
+                        ({!isCompleted ? LessThenDayMessage : CompletedLessThenDayMessage})
```

Comment:
It is better to use positive conditions. They're shorter and more readable.
```suggestion
                        ({isCompleted ? CompletedLessThenDayMessage : LessThenDayMessage})
```